website/frontend/templates/base.html

```diff
@@ -6,6 +6,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="{{ _('Official website for MusicBrainz Picard, a cross-platform music tagger written in Python.') }}">
     <meta name="author" href="{{ url_for('humans.show_humans') }}">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/js-cookie/3.0.1/js.cookie.min.js"></script>
+    <script src="https://code.jquery.com/jquery-3.7.0.js"></script>
```

Comment:
We already include jquery on all pages. Currently it is v3.6.4, maybe this needs to be updated. BUt we should not include it twice